emcc.py

```diff
@@ -764,33 +759,24 @@ def phase_setup(options, state, newargs):
       input_files.append((i, arg))
     elif arg.startswith('-L'):
       state.add_link_flag(i, arg)
-      newargs[i] = ''
```

Comment:
trying to understand what this part of the change is for...
previously we were removing the flag from the list and now aren't, so that it gets passed through to clang and clang can warn?

Reply:
Yes exactly.  If we simply allow all these flags to pass through to clang then clang can take care of issuing the warning instead of us. . which simplifies the code and paves that way for `os.execv` to be used.

Reply:
ok thanks, can you put something to that effect in the commit message?